examples/quest-3-path-options/whisperer.yaml

```diff
@@ -0,0 +1,59 @@
+entity_type: Quest
+schema_version: 1
+
+# Quest Attributes
+id: gcp-survey-path
+
+tags: [sample, life-changing, gcp]
+logo: ./quest-logo.png
+default_locale: en
+
+
+# The important part of a quest which lists all of the
+resources:
+  - type: video
+    ref_id: intro-video
+    locales:
+      en:
+        title: Welcome to GCP!
+        uri: https://youtu.be/dNW0B0HsvVs
+      es:
+        title: ¡Bienvenido a GCP!
+        uri: https://youtu.be/dNW0B0HsvVs
+  - type: video
+    ref_id: outro-video
+    locales:
+      en:
+        title: Wasn't that fun!
+        uri: https://youtu.be/dNW0B0HsvVs
+      es:
+        title: Eso fue divertido, ¿verdad?
+        uri: https://youtu.be/dNW0B0HsvVs
+
+path:
+  - content: $intro-video
+
+  - content: intro-to-gcp
+
+  - choice:
+      prompt:
+        locales:
+          en: Learn more about one of these common compute/database pairs.
+      options:
+        - path:
```

Comment:
Are nested `path`s just as powerful as top-level `path`s? And if so, if there a limit to how far they can be nested?

Reply:
I use `steps` and `path` interchangeably. I was playing around to see which I liked better; but before we finalize this, they should all be unified.